frontend/src/metabase/visualizations/echarts/cartesian/waterfall/option/axis.ts

```diff
@@ -123,16 +116,76 @@ export function getAxes(
     timelineEventsModel != null,
     renderingContext,
   );
-  const yAxis = buildMetricAxis(
-    chartModel.leftAxisModel,
+  const nameGap = getAxisNameGap(
     chartMeasurements.ticksDimensions.yTicksWidthLeft,
-    settings,
-    "left",
-    renderingContext,
-  ) as YAXisOption;
+  );
+  const range = getYAxisRange(chartModel.leftAxisModel, settings);
+  const axisType = settings["graph.y_axis.scale"] === "log" ? "log" : "value";
+
+  return {
```

Comment:
The entire `yAxis` object is built here, rather than using `buildMetricAxis`, because `buildMetricAxis` has a `valueGetter` than applies its own power transformation, which we need to avoid using for the aforementioned reason.